client/extensions/wp-super-cache/components/debug/index.jsx

```diff
@@ -68,6 +76,43 @@ class DebugTab extends Component {
 							</FormToggle>
 						</FormFieldset>
 						<div className="wp-super-cache__debug-fieldsets">
+							{ wp_cache_debug_log &&
+								<table>
```

Comment:
Heh, the old table-for-layout discussion. I thought it was sensible enough to use a column-based approach to align the filename/username labels and values, but I might be wrong :slightly_smiling_face: